doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md

```diff
@@ -2,304 +2,186 @@
+  - [Installation](#installation)
+  - [Building your application](#building-kaa-application)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-
-
-- [Java endpoint SDK](#java-endpoint-sdk)
-- [C endpoint SDK](#c-endpoint-sdk)
-    - [Install build utils and dependencies](#install-build-utils-and-dependencies)
-    - [Compile C endpoint SDK](#compile-c-endpoint-sdk)
-    - [Configure C endpoint SDK](#configure-c-endpoint-sdk)
-- [C++ endpoint SDK](#c-endpoint-sdk-1)
-    - [Install build utils and dependencies](#install-build-utils-and-dependencies)
-    - [Compile C++ endpoint SDK](#compile-c-endpoint-sdk-1)
-    - [Configure C++ endpoint SDK](#configure-c-endpoint-sdk-1)
-- [Quick way to build C/C++ endpoint SDK](#quick-way-to-build-c-c-endpoint-sdk)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+This guide describes how to use *C SDK* on linux-based systems.
 
 **Verified against:**
 
 **Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
 
-# Java endpoint SDK
-
-<!-- TODO: Fix this link when Administration-UI-guide#AdministrationUIguide-GeneratingSDK are updated -->
-To build the Java endpoint SDK, [generate](Administration-UI-guide#AdministrationUIguide-GeneratingSDK) the Java endpoint SDK in Admin UI and download the generated .jar file.
-
-# C endpoint SDK
-
-## Install build utils and dependencies
-
-Before building the C endpoint SDK, install the following components on your machine:
-
-1. Install compilers:
-    1. For automatic installation, execute the following commands:
-
-       ~~~ shel
-       $ sudo apt-get install gcc
-       ~~~
-
-    2. For manual installation of version 4.8, refer to the following example:
-
-       ~~~ shell
-       $ sudo apt-get install python-software-properties
-       $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
-       $ sudo apt-get update
-       $ sudo apt-get install gcc-4.8
-       $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 50
-       ~~~
-
-2. Install CMake utility:
-    1. For automatic installation, execute the following commands (tested on Ubuntu 14.04):
-
-       ~~~ shell
-       $ sudo apt-get install cmake
-       ~~~
-
-    2. For manual installation, refer to the following example:
-
-       ~~~ shell
-       $ wget http://www.cmake.org/files/v3.3/cmake-3.3.0-rc2.tar.gz
-       $ tar -zxf cmake-3.3.0-rc2.tar.gz
-       $ cd cmake-3.3.0-rc2/
-       $ ./configure
-       $ sudo make install
-       ~~~
-
-3. Install OpenSSL:
+## C SDK build
 
-   ~~~
-   $ sudo apt-get install libssl-dev
-   ~~~
+This section describes how to build C SDK on linux for host.
 
-4. Install CUnit:
+### Dependencies
 
-   ~~~
-   $ sudo apt-get install libcunit1-dev
-   ~~~
+To build C SDK make sure that the following components are installed:
 
-## Compile C endpoint SDK
+ - CMake (minimum required version is 2.8.8)
```

Comment:
AFAIK, minimum required version is 2.8.12. It's better not specify minimum version here, but rather link to compatibilities and dependencies page.